spec/order_total_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'order_total'
+
+describe OrderTotal do
+
+  let(:burger_description) {"Cheeseburger"}
+  let(:burger_price) {4.99}
+  let(:burger_to_s) {"#{burger_description}\t#{burger_price}"}
+  let(:burger) do double :dish,
+                  description: burger_description,
+                  price: burger_price,
+                  to_s: burger_to_s
+  end
+  let(:kebab_description) {"Kebab"}
+  let(:kebab_price) {3.00}
+  let(:kebab_to_s) {"#{kebab_description}\t#{kebab_price}"}
+  let(:kebab) do double :dish,
```

Comment:
Style/MultilineBlockLayout: Block body expression is on the same line as the block start.